.github/shellcheck-matcher.json

```
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "shellcheck",
+      "pattern": [
+        {
+          "regexp": "^(.+):(\\d+):(\\d+):\\s(note|warning|error):\\s(.*)\\s\\[(SC\\d+)\\]$",
+          "file": 1,
+          "line": 2,
+          "column": 3,
```

This would only match to warnings and errors and ignore *notes* ... that would be maybe better as less noise.
```suggestion
          "column": 3,
          "severity": 4,
```